app/assets/javascripts/epiclogger.coffee

```diff
@@ -85,7 +91,6 @@ window.EpicLogger = (->
       }
     }
     $helpfulLink = $('.helpful-link')
-    $('i.fa.fa-user').parent().attr('href').replace('1', $.auth.user.id)
```

Comment:
asta tot eu o adaugasem mai inainte, dar acum ruta pentru care o foloseam nu mai este, acum totul se intampla pe /installations. Nici macar "$('i.fa.fa-user')" nu mai avem in menu.